src/mmw/js/src/core/templates/header.ejs

```diff
@@ -13,7 +13,7 @@
                     </a>
                     <ul class="dropdown-menu">
                         <li><a href="#">Profile</a></li>
-                        <li><a href="accounts/logout">Logout</a></li>
+                        <li><a href="javascript:void(0);" class="user-logout">Logout</a></li>
```

Comment:
I hate this javascript void stuff. You don't need an href to make a link. I defer to @mmcfarland but I don't see any purpose to this and I think it looks ugly.